### ISSUE TITLE
Ingress controller errors when Kong Enterprise image tag matches bintray image tag

### ISSUE THREAD

**Issue by millermatt** [Kong/kubernetes-ingress-controller#140]
### Summary

Kong Enterprise images are supplied with tags like `0.33-1-alpine`. Creating containers in `ingress-controller.yaml` that use identical tags results in the `ingress-controller` container erroring when it starts.

**Kong Ingress controller version**
0.1.0

**Kong or Kong Enterprise version**
0.33-1-alpine

### Kubernetes version
```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.3", GitCommit:"2bba0127d85d5a46ab4b778548be28623b32d0b0", GitTreeState:"clean", BuildDate:"2018-07-26T20:40:11Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.3", GitCommit:"2bba0127d85d5a46ab4b778548be28623b32d0b0", GitTreeState:"clean", BuildDate:"2018-05-28T20:13:43Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```
### Environment

- **Cloud provider or hardware configuration**: AWS EKS
- **OS** (e.g. from /etc/os-release):  Amazon EKS-Optimized AMI
- **Kernel** (e.g. `uname -a`): 
- **Install tools**:
- **Others**:

### What happened
`ingress-controller` container fails to start when ingress-controller.yaml specifies a Kong Enterprise image with tag `0.33-1-alpine`. This is the tag used in the bintray docker registry.

Logs:
```
-------------------------------------------------------------------------------
Kong Ingress controller
  Release:    0.1.0
  Build:      git-b15730f
  Repository: git@github.com:Kong/kubernetes-ingress-controller.git
-------------------------------------------------------------------------------

W0923 22:51:18.041970       6 client_config.go:533] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
I0923 22:51:18.042235       6 main.go:204] Creating API client for https://172.20.0.1:443
I0923 22:51:18.053361       6 main.go:248] Running in Kubernetes Cluster version v1.10 (v1.10.3) - git (clean) commit 2bba0127d85d5a46ab4b778548be28623b32d0b0 - platform linux/amd64
I0923 22:51:18.057505       6 main.go:94] validated kong/kong-proxy as the default backend
F0923 22:51:18.198337       6 main.go:144] Invalid character(s) found in minor number "33-1"
```

### Expected behavior
`ingress-controller` container does not error when using official Kong Enterprise tags

### Steps To Reproduce

1. Use a tag formatted similar to `0.33-1-alpine` for the image of the `kong-migration` and `admin-api` containers in`ingress-controller.yaml`
2. Apply `ingress-controller.yaml` to the k8s cluster
3. View the logs for the `ingress-controller` container

**Comment by millermatt** [comment]
It seems to not be related to the image tag used in `ingress-controller.yaml`. I get the same error when I tag the image in my private repo as `0.33` rather than `0.33-1-alpine` and update `ingress-controller.yaml` to use the same.

It seems like the Kong API client used in the ingress-controller is probably getting the version from the Kong instance running inside one of the containers and failing to parse the result.

**Comment by millermatt** [comment]
This isn't an issue when using tag `0.33-alpine`.

**Comment by hbagdi** [comment]
This will be fixed with #141

**Comment by hbagdi** [comment]
This should be fixed with [0.1.1](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/0.1.1).
